src/Modules/PocketWaifu.cs

```diff
@@ -1649,8 +1649,8 @@ public async Task ShowThingsOnWishlistAsync([Summary("użytkownik(opcjonalne)")]
         public async Task ShowWishlistAsync([Summary("użytkownik")] SocketGuildUser usr = null,
             [Summary("czy pokazać ulubione (true/false) domyślnie ukryte, wymaga podania użytkownika")] bool showFavs = false,
             [Summary("czy pokazać niewymienialne (true/false) domyślnie pokazane")] bool showBlocked = true,
-            [Summary("czy zamienić oznaczenia na nicki?")] bool showNames = false,
-            [Summary("czy dodać linki do profili?")] bool showShindenUrl = false)
+            [Summary("czy zamienić oznaczenia na nicki? (opcjonalne)")] bool showNames = false,
```

Comment:
"opcjonalne" is unnecessary

Reply:
Again, every 'opcjonalne' from every summary in command or only this in #1652